Gemfile

```diff
@@ -0,0 +1,2 @@
+gem "qtbindings", git: "https://github.com/rock-core/qtbindings"
```

Comment:
Doesn't this mean qtbindings will be installed whether it is required or not?

Reply:
Oh my ... you're absolutely right. And it uses the git repo for everything *included* on 18.04 where it's not needed.

The only way I see to handle this would be to add gem-as-git support within the osdeps subsystem. Would you see any other ?

Reply:
Nope... I'm just not sure whether I would add support for that in osdeps files or if would rather use the Bundler management thing you implemented recently

Reply:
> would rather use the Bundler management thing you implemented recently

Not sure what you are referring to.

Reply:
I meant you could have a method like [this one](https://github.com/rock-core/autoproj/blob/25b1f1a31618fd75689a35ebcdf1dcbd7b72a250/lib/autoproj/package_managers/bundler_manager.rb#L163) to set repositories, branches

Reply:
Mmmm. Interesting. We do not get the per-OS handling, though (to avoid overriding the gem when not on Focal)

Reply:
You do if you implement it to do so :-P
I mean, one can do it in the script files using methods from the [OSPackageResolver](https://github.com/rock-core/autoproj/blob/master/lib/autoproj/os_package_resolver.rb) to get the guessed OS or you could provide an API in bundler manager for that...

Reply:
Yeah. In that case I think it will be cleaner to just add support for it from the osdeps...

Reply:
The API in `BundlerManager`, could store in the configuration something like an `only_in` setting that it would check against the guessed OS before passing it along to bundler

Reply:
Nothing against doing it in osdeps file though, I just don't know how it is going to look